retrofit-converters/protobuf-nano/README.md

```diff
@@ -0,0 +1,19 @@
+Google Nano Protocol Buffer Converter
+=====================================
+
+A `Converter` which uses [Nano Protocol Buffer][1] binary serialization.
+
+To build this module, you will need to [install][2] the Android external\_protobuf module.
+
+To use the converter, depend on the `nano` classifier:
+
+```xml
+<dependency>
+  <groupId>com.squareup.retrofit</groupId>
+  <artifactId>retrofit-converters</artifactId>
+  <classifier>nano</classifier>
```

Comment:
Looks like this isn't accurate. There's no classifier and the artifact ID is 'converter-protobuf-nano'.